src/foundry/common/data/data.mjs/activeEffectData.d.ts

```diff
@@ -10,7 +10,7 @@ interface ActiveEffectDataSchema extends DocumentSchema {
   changes: DocumentField<EffectChangeData[]> & {
     default: [];
     required: true;
-    type: [typeof EffectChangeData];
+    type: Array<typeof EffectChangeData>;
```

Comment:
what's the reason for this change? the type _is_ an array with only that one element, isn't it?